versioned_docs/version-8.7/components/modeler/web-modeler/web-modeler-settings.md

```diff
@@ -0,0 +1,70 @@
+---
+id: web-modeler-settings
+title: Web Modeler settings
+description: You can configure email notifications and process application deployment policies in the Web Modeler settings.
+---
+
+import WebModelerSettings from './img/web-modeler-settings/web-modeler-settings.png'
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+Navigate to the Web Modeler settings to configure email notifications and the process application deployment policy.
+You can find the settings by clicking on your user icon in the top right corner of the Web Modeler and selecting **Settings**.
+
+<img src={WebModelerSettings} alt="Web Modeler settings" />
+
+## Email notifications
+
+Configure the projects for which you will receive email notifications when a collaborator mentions you in a comment.
```

Comment:
How do you do this? Are there particular UI elements I should be looking for?

Reply:
There is a toggle per project. It's quite obvious if you navigate to the settings page. Does this require any changes?

Reply:
It's what is shown in this screenshot: 
![image](https://github.com/user-attachments/assets/ba3f1820-ffc1-4681-bd0e-71bab2a6711b)

Reply:
Committing a suggestion 👍